### PR TITLE
Added devfile.yaml for workspace-loader

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

This `devfile.yaml` provides commands useful for developing `workspace-loader` in Che Theia.

Signed-off-by: Oleksii Kurinnyi <okurinny@redhat.com>

### CODE REVIEW

File: workspace-loader/.devfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          protocol: http
+          public: "true"
+    mountSources: true
+    memoryLimit: 1Gi
+commands:
+  - name: '[workspace loader] install dependencies'
+    actions:
+      - type: exec
+        component: ws-loader-dev
+        command: cd /projects/che/workspace-loader && yarn
```
Comment (AndrienkoAleksandr) [inline workspace-loader/.devfile]: Why Do you need `cd` if you defined  `workdir`? And the same for each command...

**Comment by AndrienkoAleksandr** [comment]
@akurinnoy apply please type scipt intelligence:
 ```
components:
 - alias: typescript-plugin
    type: chePlugin
    id: che-incubator/typescript/1.30.2
```

**Comment by monaka** [comment]
Why do you use `.devfile` instead of `devfile.yaml` ? (refs https://github.com/eclipse/che/pull/12965 )

**Review by AndrienkoAleksandr — APPROVED** [review]

**Comment by akurinnoy** [comment]
@monaka thanks for your comment! I've updated title and commit message.

**Pull request merged**